README.md

```diff
@@ -9,23 +9,159 @@ An engineer working for a [CSE](./CSE.md) project...
 
 This is the playbook. You are invited to contribute via standard git pull requests to GitHub.
 
-## The Basics
-
-* [Source Control](./Engineering/SourceControl.md)
-* [Unit Testing](./Engineering/UnitTesting.md)
-* [Continuous Integration / Continuous Delivery](./Engineering/CICD.md)
-* [Code Reviews](./Engineering/CodeReviews.md)
-* [Retrospectives](./Engineering/Retrospectives.md)
-* [Engineering Practices And Principles](./Engineering/PracticiesAndPrinciples.md)
-
-## Seeking Contributions for
-
-- Component Tests
-- DevOps
-    - [Component Versioning](./Engineering/ComponentVersioning.md)
-    - [Logging and Monitoring](./Engineering/DevOpsLogging.md)
-    - Performance
-    - Telemetry
-- Security
-    - Threat Model
-- Regulatory Compliance
+## Outline
+
+The following outlines each section and what content would be included within each section.
+
+### Goals
+
+The proposed outline is intended to accomplish the following goals
+
+1. Author as idea-to-delivery value stream narrative from an engineering perspective.
+2. Organize for quick reference and discoverability.
+3. Extensible heirarchy to allow teams to share deep subject matter expertise.
+
+### Working Heirarchy
```

Comment:
@fnocera mentioned possibly having a TOC somewhere for all ML-related content, which would otherwise be peppered throughout SE docs (either inside or as new files). Capturing that comment here so we don't lose it.

Reply:
I dont want to block the outline on this particular point at the moment. I think as we get more content added a reasonable place will emerge to pull together this content. For the moment though this is a bit counter to the "lets tell a story" structure we wanted to use initially.

If we were using an actual wiki (i.e. Confluence) i would typically use tags/labels for this type of thing but im not sure such a construct exists for markdown docs in a github repo.